distributions/test/itests/src/test/java/org/codice/ditto/replication/admin/query/ITReplicationQuery.java

```diff
@@ -52,23 +64,52 @@
 )
 @ServiceAdmin
 @ConfigurationAdmin
+// keeps solr from changing the password so we can use "admin:admin" with basic auth to contact solr
+@SetSystemProperty(key = "solr.attemptAutoPasswordChange", value = "false")
```

Comment:
✏️ That is probably something that should be added to the `InstallDistributionExtension` class in DDF